src/main/@orb.yml

```diff
@@ -1,2 +1,5 @@
 version: 2.1
-description: Detect bugs and vulnerabilities in your repository. This orb is created by SonarSource (https://www.sonarsource.com/), the source is available at https://github.com/SonarSource/sonarcloud-circleci-orb
\ No newline at end of file
+description: Detect bugs and vulnerabilities in your repository.
+display:
+  home_url: https://www.sonarsource.com/
```

Comment:
SonarSource is the editor, SonarCloud is the service. So don't you think having `https://sonarcloud.io` would be more relevant?